docs/docusaurus/docs/glossary.md

```diff
@@ -25,8 +25,6 @@ import CLIRemoval from '/docs/components/warnings/_cli_removal.md'
 
 [**Data Assistant:**](./terms/data_assistant.md) A utility that asks questions about your data, gathering information to describe what is observed, and then presents Metrics and proposes Expectations based on the answers.
 
-[**Data Connector:**](./terms/data_connector.md) Provides the configuration details based on the source data system which are needed by a Datasource to define Data Assets.
```

Comment:
Is it worth keeping the entry in the glossary but saying something like "The DataConnectors are now an implementation detail the users don't need to care about"